lib/widgets/child_wallet.dart

```diff
@@ -38,6 +39,26 @@ class ChildWallet extends StatelessWidget {
 
 }
 
+class ChildPointsChip extends StatelessWidget {
+	// TODO Passing currency and using it's icon & color
+	final int quantity;
+
+	ChildPointsChip({
+		@required this.quantity
+	});
+
+	@override
+	Widget build(BuildContext context) {
+		return Chip(
+			backgroundColor: lighten(AppColors.diamondColor, 0.48), 
+			avatar: SvgPicture.asset('assets/image/currency/diamond.svg', width: 24, fit: BoxFit.cover),
+			label: Text((quantity ?? 0).toString(), style: TextStyle(color: AppColors.diamondColor, fontWeight: FontWeight.bold)),
+			visualDensity: VisualDensity.compact
+		);
+	}
+
+}
+
 class ChildCustomHeader extends StatelessWidget {
```

Comment:
Why is this widget in the wallet file anyway? It belongs with the header stuff.

Reply:
Wallet file deleted. Child custom header in app_header file now.
Removed additional wallet pop-up for now.